### DISCUSSION TITLE
"Developer Mode" - How to disable?

### DISCUSSION THREAD

**Discussion by coolduke666** [magicbug/Cloudlog#2460] [General]
installed Cloudlog in and Debian VM and have "Developer Mode" enabled - please information how and where to disable the developer mode?

best

**Comment by phl0** [comment]
Edit index.php and change line 57 to:

```
define('ENVIRONMENT', 'production');
```

**Comment by coolduke666** [comment]
Worked like a charm... thanks for the support

Florian (DF2ET) ***@***.***> schrieb am Sa., 2. Sep. 2023,
09:50:

**Comment by Johnty27** [comment]
Guys, where do I find index.php?

Lots of .php files in the /config/data volume but the only file called "index" is a html file.

Am I looking in the wrong place??

Install is on my Openmediavault server using Docker Compose.

John (G7SSE)

**Reply by phl0** [reply]
In the (web)root folder of your instance.

**Reply by rayslinky** [reply]
It would be nice to move this to the config file.

**Reply by Johnty27** [reply]
Found it, thanks.

John (G7SSE)

**Comment by resnikov** [comment]
Where do you disable this in the docker version?

**Reply by rayslinky** [reply]
Ideally this config should be moved to the config.php.

Currently, I map in a copy of index.php, so I can override. Unf, this means you have to watch for changes to index.php.

**Reply by petrkr** [reply]
best will be take it from PHP environment variable.
